components/scream/src/physics/p3/CMakeLists.txt

```diff
@@ -12,6 +12,7 @@ set(P3_SRCS
   p3_ic_cases.cpp
   micro_p3_iso_c.f90
   micro_p3_iso_f.f90
+  ${SCREAM_BASE_DIR}/../cam/src/physics/cam/scream_abortutils.F90
```

Comment:
Why putting `scream_abortutils.F90` _outside_ of the scream folder? Why not just put it in `scream/src/share`?

Reply:
Thanks @bartgol for your review. I consider this a throw away file (like other F90 codes), so I decided to put it in `cam` tree. I can move it to `scream/src/share`, if it is more appropriate.

Reply:
Ok. If this file is to be compiled only in full E3SM, then you can leave it there, if you want.